### PR TITLE
add pv-migrate plugin

### PR DESCRIPTION
**Pull request by utkuozdemir**
Signed-off-by: Utku Ozdemir <uoz@protonmail.com>

Follow-up on the question: https://github.com/kubernetes-sigs/krew-index/issues/1403

Tested using `kubectl krew install --manifest` on Windows, MacOS and Raspberry Pi.

`krew-release-bot` is enabled for the future releases.

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

**Review by ahmetb — COMMENTED** [review]
Overall good. Minor comment. 
/approve

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

File: plugins/pv-migrate.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      sha256: 1b9365e22e39e99d90b2d2c330cd7d9dcbee5833e27bb4cbade6791a689a9bea
+      bin: pv-migrate
+    - selector:
+        matchLabels:
+          os: windows
+          arch: amd64
+      uri: https://github.com/utkuozdemir/pv-migrate/releases/download/v0.8.1/pv-migrate_v0.8.1_windows_x86_64.zip
+      sha256: 4ce5c61aee68687ad5f79b9fe3ef7fe0c76140d21cace21261ebc998035156e4
+      bin: pv-migrate.exe
+  shortDescription: CLI tool to easily migrate Kubernetes persistent volumes
```
[the remark this answers is not in the document]
Comment (utkuozdemir) [reply plugins/pv-migrate.yaml]: thanks, updated

**Comment by ahmetb** [comment]
/cc @corneliusweig 
if you'd like to weigh in again, but this lgtm.

**Pull request merged**